### DISCUSSION TITLE
Why there is no centos

### DISCUSSION THREAD

**Discussion by lbhnrg2021** [microsoft/WSL#5962] [General]
Why is there no centos subsystem

**Comment by anwesh-b** [comment]
Download the zipped Centos file from [here](https://github.com/yuk7/CentWSL/releases)
Unzip the file and run `CentOS.exe`
This will complete the installation.
Again, run the `CentOS.exe` and you are ready to go

**Comment by lbhnrg2021** [comment]
> Download the zipped Centos file from [here](https://github.com/yuk7/CentWSL/releases)
> Unzip the file and run `CentOS.exe`
> This will complete the installation.
> Again, run the `CentOS.exe` and you are ready to go

This is third-party and there is a problem,  docker cannot start

**Comment by craigloewen-msft** [comment]
CentOS is a great request! The decision to put CentOS in the Microsoft Store is up to the owners of the distros themselves (RedHat). We have had some discussions with them about adding CentOS and other RedHat distros to the store, and I'll be sure to make sure we represent that there are many folks in our community who would love to see those distros there!

**Reply by haosmos** [reply]
> CentOS is a great request! The decision to put CentOS in the Microsoft Store is up to the owners of the distros themselves (RedHat). We have had some discussions with them about adding CentOS and other RedHat distros to the store, and I'll be sure to make sure we represent that there are many folks in our community who would love to see those distros there!

It would be great to be able to use Fedora in wsl2. Fedora is one of the most actively developing distributions.

**Reply by Biswa96** [reply]
If you have money to waste then https://www.microsoft.com/store/productId/9N6GDM4K2HNC. Otherwise there is a trick which waste some of your time.

**Reply by lbhnrg2021** [reply]
Sorry, I hope open source

**Comment by luisdev** [comment]
It seems like the CentOS platform is now dead:
https://arstechnica.com/gadgets/2020/12/centos-shifts-from-red-hat-unbranded-to-red-hat-beta/

So is there any point in trying to run CentOS under WSL anymore, is there?
